internal/service/federatedsettingsorgconfig/data_source_federated_settings_connected_org_test.go

```diff
@@ -28,8 +26,7 @@ func TestAccFederatedSettingsOrgDS_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resourceName, "federation_settings_id"),
 					resource.TestCheckResourceAttrSet(resourceName, "role_mappings.#"),
 					resource.TestCheckResourceAttrSet(resourceName, "identity_provider_id"),
-					resource.TestCheckResourceAttrSet(resourceName, "org_id"),
-					resource.TestCheckResourceAttr(resourceName, "identity_provider_id", "0oad4fas87jL5Xnk1297"),
```

Comment:
can't be known?

Reply:
Replaced `org_id` with a more specific check (value of `org_id` ).
Removed `identity_provider_id` as it depends on the environment. Could add an env-var for the specific value but seems fragile and overkill 😅